trivia.py

```diff
@@ -35,10 +35,10 @@ def assemble_db(self, removeDisqualifyingCategories=True):
         """Consolidate JSON-oriented Quester/Answer File into a Dictionary"""
         # rather than read the file multiple times, we convert to a dictionary first
         db = {}
-        self.logger.debug("type(self.raw)=" + str(type(self.raw)))
+        #self.logger.debug("type(self.raw)=" + str(type(self.raw)))
```

Comment:
why are these commented out? seem like they are useful

Reply:
i put them back in - i originally removed them simply because they were so noisy